### PR TITLE
documentation: Instructions for local development

### PR DESCRIPTION
**Pull request by Soyn**
Update the documentation for local development.

#2165

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -97,6 +97,9 @@ For contributing features and fixes you can quickly get an environment running u
     1. `SLACK_SECRET` (this is called "Client Secret" in Slack admin)
 1. Configure your Slack app's Oauth & Permissions settings 
     1. Add `http://localhost:3000/auth/slack.callback` as an Oauth redirect URL
+       1. In the latest Slack settings, the `http//*` cannot be added for security. For local development, you can use a tool like `ngrok` or a package like `mkcert` and I've included some links below.
+          1. [ngrok](https://ngrok.com)
+          1. [How to use HTTPS for local development](https://web.dev/how-to-use-local-https/)
     1. Ensure that the bot token scope contains at least `users:read`
```
Comment (tommoor) [inline README.md:103]:
```suggestion
    1. Slack recently prevented the use of `http` protocol for localhost. For local development, you can use a tool like [ngrok](https://ngrok.com) or a package like `mkcert`. ([How to use HTTPS for local development](https://web.dev/how-to-use-local-https/))
    1. Add `https://my_ngrok_address/auth/slack.callback` as an Oauth redirect URL
    1. Ensure that the bot token scope contains at least `users:read`
```

Comment (tommoor) [reply README.md:103]: Prefer not to use the "I" pronoun in documentation

**Comment by Soyn** [comment]
updated the doc by suggestion.

**Comment by tommoor** [comment]
Please see my suggestion again, sorry the instructions still don't make total sense as the first step is `http://localhost:3000` which no longer works, there's also an extra level of nesting – at three levels of nesting it becomes confusing to read.

**Comment by Soyn** [comment]
fixed.

**Pull request merged**